assets/keymaps/default-macos.json

```diff
@@ -604,6 +604,12 @@
       "escape": "chat_panel::CloseReplyPreview"
     }
   },
+  {
+    "context": "ChatPanel > MessageEditor",
+    "bindings": {
+      "escape": "chat_panel::CancelEditMessage"
+    }
+  },
```

Comment:
@RemcoSmitsDev we need a corresponding change in the linux keymap too.

Reply:
Actually, thinking more about this, you may be able to listen on `editor::Cancel` here instead of adding a new action (see how the project panel does it).

Reply:
If that works, that seems better to me! (One fewer keybinding for people to manage)